tests/views/test_collection_exercise.py

```diff
@@ -873,7 +880,9 @@ def test_no_upload_collection_instrument_when_bad_extension(self, mock_request,
         post_data = {"ciFile": (BytesIO(b"data"), "064_201803_0001.html"), "load-ci": ""}
         mock_details.return_value = formatted_collection_exercise_details
 
-        response = self.client.post(f"/surveys/{short_name}/{period}", data=post_data, follow_redirects=True)
+        response = self.client.post(
+            f"/surveys/{short_name}/{period}/view-sample-ci", data=post_data, follow_redirects=True
+        )
 
         self.assertEqual(response.status_code, 200)
         self.assertNotIn("Collection instrument loaded".encode(), response.data)
```

Comment:
There seems to be again chunks of the same code, we should rationalise so this file isn't massive

Reply:
To be done in another card